linux_os/guide/system/auditing/auditd_configure_rules/audit_execution_selinux_commands/audit_rules_execution_chcon/rule.yml

```diff
@@ -46,6 +47,10 @@ references:
     srg: SRG-OS-000392-GPOS-00172,SRG-OS-000463-GPOS-00207,SRG-OS-000465-GPOS-00209
     vmmsrg: SRG-OS-000463-VMM-001850
     stigid@rhel7: RHEL-07-030580
+    stigid@sle12: SLES-12-020630
```

Comment:
@yuumasato do you think that this change in the OVAL can negatively affect other products?

https://github.com/ComplianceAsCode/content/pull/6631/files#diff-cce2ac5d738ae7a0d1e45f31e641ec1c63a95e80581797eaa62db093ce9b1dab

The addition regex clause is marked as optional `?`. So in my opinion, it shouldn't affect, but it might have side effects to the rule description.

Reply:
Actually, I I think there is a change in behavior  when`-F perm=x` is present, as the absence of field `perm` means all auditable behaviors are recorded.
https://github.com/ComplianceAsCode/content/pull/5956#issuecomment-666237726

So the change needs to be wrapped around a Jinja `if`, to preserve behavior for RHEL.

Reply:
OK, will add to description and ocil:
+    {{% if product in ["sle12", "sle15"] %}}
+    <pre>-a always,exit -F path=/usr/bin/chcon -F perm=x -F auid&gt;={{{ auid }}} -F auid!=unset -F key=privileged</pre>
+    {{% else %}}
     <pre>-a always,exit -F path=/usr/bin/chcon -F auid&gt;={{{ auid }}} -F auid!=unset -F key=privileged</pre>
+    {{% endif %}}